app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = len(friends)
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+            masks_to_buy -= 1
```

Comment:
May better set `masks_to_buy` as `0` and when excepted `NotWearingMaskError`  add +1